### PR TITLE
[Refactor] Merge Release to Develop

### PR DESCRIPTION
**Pull request by jpeng-ms**
## Purpose
This PR merges release branch back to develop to make sure we are not branching off too far and avoid any merge conflicts later on.
## Does this introduce a breaking change?

```
[ ] Yes
[X] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[X] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```
N/A

## What to Check
N/A

## Other Information
N/A

### CODE REVIEW

**Comment by jpeng-ms** [comment]
Do not click "update branch":

image

**Review by palatter — APPROVED** [review]

File: AzureCommunicationUI/sdk/AzureCommunicationUICalling/Sources/Localization/zh-Hans-CN.lproj/Localizable.strings [hunk 1]
```diff
@@ -100,3 +100,4 @@
 "AzureCommunicationUICalling.SnackBar.Text.ErrorResumeCallSubTitle"="其他应用正在使用你的麦克风";
 "AzureCommunicationUICalling.SnackBar.Text.ErrorCallDenied"="你被拒绝接听电话。";
```
Comment (palatter) [inline AzureCommunicationUI/sdk/AzureCommunicationUICalling/Sources/Localization/zh-Hans-CN.lproj/Localizable.strings:101]: Is there a reason this locale got fewer additions?

Comment (jpeng-ms) [reply AzureCommunicationUI/sdk/AzureCommunicationUICalling/Sources/Localization/zh-Hans-CN.lproj/Localizable.strings:101]: Good catch! Let me check with Leah

Comment (Leah-Xia-Microsoft) [reply AzureCommunicationUI/sdk/AzureCommunicationUICalling/Sources/Localization/zh-Hans-CN.lproj/Localizable.strings:101]: @palatter There was a PR of localization merged into develop before cutting the release branch. However, back then there's some issue with the pipeline so the translation was not done for some languages. Later the fix went directly into release branch. So when we merge back from release to develop, the there's this difference in terms of additions.

**Review by Leah-Xia-Microsoft — APPROVED** [review]

File: AzureCommunicationUI/sdk/AzureCommunicationUICalling/Sources/Localization/zh-Hans-CN.lproj/Localizable.strings (hunk 1 above)
Comment (palatter) [reply AzureCommunicationUI/sdk/AzureCommunicationUICalling/Sources/Localization/zh-Hans-CN.lproj/Localizable.strings:101]: Sounds good, thanks!

**Pull request closed without merging**